### ISSUE TITLE
PDF form fill with unicode string - The character is not supported by the font exception

### ISSUE THREAD

**Issue by giaur500** [syncfusion/flutter-widgets#846]
Hello,

I need to fill pdf form with unicode string, cod to do that:

```
final byteData = await rootBundle.load('assets/reports/report1.pdf');
    final document = PdfDocument(inputBytes: byteData.buffer.asUint8ClampedList());
    final form = document.form;
    final fields = form.fields;

    (fields[0] as PdfTextBoxField).text = 'łł';

    form.flattenAllFields();

    final List<int> bytes = await document.save();

    document.dispose();
```

But I'm getting an exception:
> [ERROR:flutter/lib/ui/ui_dart_state.cc(209)] Unhandled Exception: Invalid argument (The character is not supported by the font.)

How to fill form with non latin - unicode characters?

**Comment by GowthamrajKumar25** [comment]
Hi glaur500,

We were able to get the reported exception with provided details on our end. Currently, we are analyzing on this and we will update the further details on August 16th 2022. 

Regards,
Gowthamraj K

**Comment by GowthamrajKumar25** [comment]
Hi glaur500,

We have confirmed that the issue with “**Exception occurs when filling the form field with Unicode text in Flutter PDF**” is a defect. The fix for this issue will be available in our weekly release on August 23, 2022.

Kindly use the below feedback link to track the status of the reported bug,

https://www.syncfusion.com/feedback/37053/exception-occurs-when-filling-the-form-field-with-unicode-text-in-flutter-pdf

Please let us know if you need any further assistance in this.

Regards,
Gowthamraj K

**Comment by GowthamrajKumar25** [comment]
Hi giaur500,

The fix for the reported issue “Exception occurs when filling the form field with Unicode text in Flutter PDF” has been included in our weekly release  v20.2.45.

Kindly use the following link to get our latest package version,
https://pub.dev/packages/syncfusion_flutter_pdf/versions/20.2.45

Please let us know if you need any further assistance in this.

Regards,
Gowthamraj K
